Vagrantfile

```diff
@@ -0,0 +1,62 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+$script = <<SCRIPT
+function Add-Path($path) {
+  if(!$env:path.Contains($path)) {
+    $new_path = "$env:PATH;$path"
+    $env:PATH=$new_path
+    [Environment]::SetEnvironmentVariable("path", $new_path, "Machine")
+  }
+}
+
+iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
+choco install ruby -x86 -y
+Add-Path "C:/tools/ruby21/bin"
+
+choco install ruby2.devkit -x86 -y
+Add-Content -Value "- c:/tools/ruby21" -Path "c:/tools/DevKit2/config.yml"
+cd c:/tools/DevKit2
+ruby dk.rb install -f
+
+gem install bundler
+
+choco install wixtoolset -y
+Add-Path "C:/Program Files (x86)/Wix Toolset v3.9/bin"
+
+choco install windows-sdk-8.1 -y
+Add-Path "C:/Program Files (x86)/Windows Kits/8.1/bin/x86"
+
+choco install git -y
+Add-Path "C:/Program Files/Git/cmd;C:/Program Files/Git/usr/bin"
+
+git config --global user.email "vagrant@example.com"
+git config --global user.name "vagrant"
+
+mkdir c:/omnibus-chef
+Copy-Item c:/vagrant/* c:/omnibus-chef -recurse -force
+
+cd c:/omnibus-chef
+bundle install
+bundle update omnibus-software
+bundle exec omnibus build chefdk
+SCRIPT
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "mwrock/Windows2012R2"
+  config.vm.provision "shell", inline: $script
+  config.vm.guest = :windows  
+
+  config.vm.provider "hyperv" do |hv|
+    hv.ip_address_timeout = 240
+    hv.memory = 2048
+    hv.cpus = 2
+  end
+
+  config.vm.provider :virtualbox do |vb|
+    vb.gui = true
+    vb.memory = 2048
+    vb.cpus = 2
+    vb.network "forwarded_port", guest: 5985, host: 55985
+  end
+end
```

Comment:
Why are we introducing an `Vagrantfile` here? All other platforms (including Windows) build with the included `.kitchen.yml`.